src/trigger.ts

```diff
@@ -213,13 +215,15 @@ function createTriggers(targetEngine: IEngine) {
 					continue
 
 				const intersecting = areTriggersIntersecting(shapeWorldPos.get(entity)!, trigger, shapeWorldPos.get(otherEntity)!, otherTrigger)
-				if (intersecting)
-					collisions.add(otherEntity)
+				if (intersecting) { }
```

Comment:
it should read.  you open then closed the if immediately :)


`if (intersecting) {
    if (collisions === EMPTY_IMMUTABLE_SET) collisions = new Set()
    collisions.add(otherEntity)
}`

Reply:
haha yikes, I must have messed it up while refactoring
good catch!